kubernetes-services/work-scheduler/config/service-template.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: harmony-work-scheduler
+  labels:
+    name: harmony
+spec:
+  replicas: 1
```

Comment:
Is there any reason we might want to run more than one? Or maybe this is only used locally so it's fine to hardcode and not use a variable here.

Reply:
this is only used locally